README.md

```diff
@@ -39,15 +39,15 @@ In order to transform Unreal from a single server engine to a distributed model,
 We have released the GDK for Unreal this early in development because we want your feedback. Please come and talk to us about the software and the documentation via: [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) - [Forums](https://forums.improbable.io/) - [GitHub issues in this repository](https://github.com/spatialos/UnrealGDK/issues).
 
 ## Where to get the GDK and related projects
-The GDK and its example projects are available on GitHub.
+The GDK and its related projects are available on GitHub.
 * [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
 * [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK)
-**NOTE:** This link may give you a 404.
+**NOTE:** This link may give you a 404. 
```

Comment:
We also mention the 404 issue above (under Unreal Engine changes), but there we link directly to the GDK docs on the main docs site, instead of explaining inline. Do we want to do the same here? (Or explain inline in both places?)

It's a minor point though - probably fine to stay as is.

Reply:
Yeah not sure to be honest. I'll leave it as is right now.